app/controllers/orders_controller.rb

```diff
@@ -29,6 +34,12 @@ def create
     # send mail
   end
 
+  def show
+    @order = Order.find_by id: params[:id]
```

Comment:
`@order` bị nul thì sao anh ơi

Reply:
anh fix rồi nha. redirect về root và alert not found.